cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/BusinessMetadataDataset.java

```diff
@@ -341,22 +480,26 @@ private String getTargetType(Id.NamespacedId namespacedId) {
 
   private String getMetadataKey(String type, byte[] rowKey) {
     MDSKey.Splitter keySplitter = new MDSKey(rowKey).split();
-    // The rowkey is [targetType][targetId][key], so skip the first few strings.
+    // The rowkey is [targetType][targetId][metadata-type][key], so skip the first few strings.
```

Comment:
Need to update this comment and others to make sure it reflect the right MDSKey format.